its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -457,4 +461,19 @@ private static List<String> extractCeTaskIds(BuildResult buildResult) {
       .map(s -> s.substring(s.lastIndexOf("=") + 1))
       .collect(Collectors.toList());
   }
+
+  private static Command initCommandEnvironment(Command command, List<EnvironmentVariable> environmentVariables) {
+    var buildDirectory = environmentVariables.stream().filter(x -> x.getName() == AzureDevOps.AGENT_BUILDDIRECTORY).findFirst();
+    if (buildDirectory.isPresent()) {
```

Comment:
Isn't this always true because of `setEnvironmentVariable(AzureDevOps.AGENT_BUILDDIRECTORY, "");` in the ScannerCommand ctor?
https://github.com/SonarSource/sonar-scanner-msbuild/pull/2377/files#diff-26a9c5c6996b66362c959d2de0eedb34e11cf8a52d5bced79cd8bf4e3bb0d98dR66